tests/develop/xdebug_time_index.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+xdebug_time_index()
+--INI--
+xdebug.mode=develop
+--FILE--
+<?php
+usleep( 250000 );
+
+$rq = $_SERVER['REQUEST_TIME_FLOAT'];
```

Comment:
just capture another `microtime(true)` before usleep, this test should not test if php globals work correctly/are set, but instead only if xdebug time works

Reply:
xdebug_time_index starts at the start of the request, which is closer to `$_SERVER['REQUEST_TIME_FLOAT'];` than `microtime()` in the code